### ISSUE TITLE
Conflict with vim-nerdtree-tabs

### ISSUE THREAD

**Issue by cheap-glitch** [PhilRunninger/nerdtree-visual-selection#6]
When opening several files in new tabs, vim produces the following errors for each file:
```
E127: Cannot redefine function NERDTree_Open: It is in use
E127: Cannot redefine function <SNR>90_ProcessSelection: It is in use
```
The errors don't prevent the files from being opened correctly, but as they are displayed in the log one by one, they do slow down the process significantly.

I only get these errors when using the plugin [vim-nerdtree-tabs](https://github.com/jistr/vim-nerdtree-tabs). When I remove it, the errors go away.

**Comment by PhilRunninger** [comment]
I'm not seeing this behavior. You'll need to provide your settings and complete instructions to reproduce the error. I suspect this is a problem with `vim-nerdtree-tabs`, and it's trying to source the `nerdtree-visual-selection` files again, while their functions are already running.

**Comment by cheap-glitch** [comment]
You can reproduce the bug with this minimal `.vimrc`:
````vim
call plug#begin('~/.vim/plugged')

Plug 'PhilRunninger/nerdtree-visual-selection'
Plug 'jistr/vim-nerdtree-tabs'
Plug 'scrooloose/nerdtree'

call plug#end()
````
I use vim-plug as my plugin manager, but I don't think it affects anything.

**Comment by PhilRunninger** [comment]
Sorry for the late reply. I've been busy lately. I wonder if  the order matters. What happens if you put `scrooloose/nerdtree` first in your list of plugins?

**Comment by cheap-glitch** [comment]
> Sorry for the late reply. I've been busy lately.

No problem :)

> I wonder if the order matters. What happens if you put scrooloose/nerdtree first in your list of plugins?

I wondered about it too so I tried a few possible orders, but it doesn't seem to change anything.

**Comment by PhilRunninger** [comment]
I reproduced the error thusly:
```
:NERDTreeTabOpen
Vjjj
t
```

**Comment by PhilRunninger** [comment]
Fixed in f940a25e0672922cf3a1fb41e1c1a40ccea9e03f.
